setup.py

```diff
@@ -113,7 +114,7 @@ def alltests():
           'Topic :: Internet :: WWW/HTTP',
           'Framework :: Zope3',
           ],
-      url='http://pypi.python.org/pypi/zope.container',
+      url='http:///zopecontainer.readthedocs.io',
```

Comment:
@icemac [also prefers the documentation link rather than the github link](https://github.com/zopefoundation/zope.size/pull/3#discussion_r92557104).